src/api/shelter/admin/essential-info.ts

```diff
@@ -0,0 +1,27 @@
+import api from '@/api/instance';
+
+export type Address = {
+  address: string;
+  addressDetail: string;
+  postalCode: string;
+  latitude: number;
+  longitude: number;
+};
+export interface ShelterEssentialInfo {
+  name: string;
+  phoneNumber: string;
+  description: string;
+  address: Address;
+}
+
+export type ShelterEssentialInfoPayload = ShelterEssentialInfo;
+export type PutResponse = {
+  shelterId: number;
+  shelterUserId: number;
+};
+
+export const put = async (payload: ShelterEssentialInfoPayload) => {
+  return await api
+    .put('shelter/admin/essential-info', { body: JSON.stringify(payload) })
+    .json<PutResponse>();
```

Comment:
ky json 옵션을 이용하면 객체를 자동으로 JSON.stringify() 처리해준다고 합니다~!

[ky 깃허브 본문 내용](https://github.com/sindresorhus/ky)

json

Shortcut for sending JSON. Use this instead of the body option. Accepts any plain object or value, which will be JSON.stringify()'d and sent in the body with the correct header set.

Reply:
오 적용하겠습니다 감사합니다!